resources/todomvc/architecture-examples/angular/src/app/todo-list/todo-list.component.html

```diff
@@ -3,7 +3,7 @@
         <input class="toggle-all" type="checkbox" (change)="toggleAll($event)" [checked]="!activeTodos.length" />
         <label class="toggle-all-label" htmlFor="toggle-all"> Toggle All Input </label>
     </div>
-    <ul class="todo-list">
+    <ul class="todo-list show-priority">
```

Comment:
I don't see where this new class is used in the CSS. Am I missing something?

Reply:
It is used here https://github.com/WebKit/Speedometer/blob/3322fd2ba8dc6f718250eba4a2b9dd57e4305598/resources/todomvc/big-dom-generator/utils/app.css#L424

Reply:
and now I see it, thanks!